src/sentry/monitors/endpoints/organization_monitors.py

```diff
@@ -77,17 +77,31 @@ def get(self, request: Request, organization: Organization) -> Response:
         else:
             environments = list(Environment.objects.filter(organization_id=organization.id))
 
-        # sort monitors by top monitor environment
-        monitorenvironment_top_status = Subquery(
-            MonitorEnvironment.objects.filter(
-                monitor__id=OuterRef("id"), environment__in=environments
+        # sort monitors by top monitor environment, then by last check-in
```

Comment:
The comment here seems slightly outdated now no?

Reply:
outdated how? it's slightly confusing but i can diagram it out

Reply:
It doesn't indicate that this is sorting disabled monitors to the bottom right